Reason: '{,1}' is not valid in regular expressions, it should be '{0,1}'.
Busybox sed does not accept the former.
Upstream: no

diff --git a/rdnss/merge-hook.in b/rdnss/merge-hook.in
index 383a57c..00ddcb2 100644
--- a/rdnss/merge-hook.in
+++ b/rdnss/merge-hook.in
@@ -38,7 +38,7 @@ myresolvconf="$localstatedir/run/rdnssd/resolv.conf"
 
 # These should be POSIX-compliant BREs
 RE_NSV4='^nameserver  *\([0-9]\{1,3\}\.\)\{3,3\}[0-9]\{1,3\} *$'
-RE_NSV4OR6='^nameserver  *[a-fA-F0-9:\.]\{1,46\}\(%[a-zA-Z0-9]\{1,\}\)\{,1\} *$'
+RE_NSV4OR6='^nameserver  *[a-fA-F0-9:\.]\{1,46\}\(%[a-zA-Z0-9]\{1,\}\)\{0,1\} *$'
 
 # Count how many IPv6 nameservers we can fit
 
